src/styles/main.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
```

Comment:
Would be great to move all the styles of `stars` block to the separate file

Reply:
Got it, thanks)